lib/pdk/module/convert.rb

```diff
@@ -68,25 +72,30 @@ def needs_bundle_update?
       def stage_changes!
         metadata_path = 'metadata.json'
 
-        PDK::Module::TemplateDir.new(template_uri, nil, false) do |templates|
+        PDK::Module::TemplateDir.new(template_uri, nil, true) do |templates|
```

Comment:
The `PDK::Module::TemplateDir` constructor seems like a good candidate for keyword arguments. Not necessarily now though.